extensions/adobe/experience/campaign-experienceevent.schema.json

```diff
@@ -17,91 +17,414 @@
     "https://ns.adobe.com/experience/decisioning/decisionevent"
   ],
   "meta:extends": [
-    "https://ns.adobe.com/xdm/context/experienceevent-advertising",
     "https://ns.adobe.com/xdm/context/experienceevent-application",
     "https://ns.adobe.com/xdm/context/experienceevent-channel",
-    "https://ns.adobe.com/xdm/context/experienceevent-commerce",
     "https://ns.adobe.com/xdm/context/experienceevent-directmarketing",
-    "https://ns.adobe.com/xdm/context/experienceevent-environment-details",
-    "https://ns.adobe.com/xdm/context/experienceevent-marketing",
-    "https://ns.adobe.com/xdm/context/experienceevent-media",
-    "https://ns.adobe.com/xdm/context/experienceevent-profile-stitch",
-    "https://ns.adobe.com/xdm/context/experienceevent-search",
-    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
-    "https://ns.adobe.com/xdm/context/experienceevent-technical-details",
-    "https://ns.adobe.com/xdm/context/experienceevent-web",
-    "https://ns.adobe.com/experience/campaign/experienceevent-all",
     "https://ns.adobe.com/experience/decisioning/proposition",
-    "https://ns.adobe.com/experience/target/experienceevent-shared",
-    "https://ns.adobe.com/experience/profile/experienceevent-shared",
-    "https://ns.adobe.com/experience/implementations-ext",
-    "https://ns.adobe.com/xdm/context/experienceevent-enduserids"
+    "https://ns.adobe.com/xdm/context/experienceevent-environment-details"
   ],
   "definitions": {
     "campaign-experienceevent": {
-      "properties": {}
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/message": {
+          "title": "A single message sent to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
```

Comment:
When we removed experienceevent-all mixin from this, I think we have missed adding profileSnapshot object from there to this mixin. We should include it too.

Reply:
@aarjain what was the need for profileSnapshot?  I had in my notes this could be removed as its not ever in your payload.  Can you confirm?

Reply:
It is used for passing the profile fields at the time of creation of experience event for the profile.

Reply:
@eknee : Adding on to what @sawhneyh said, you can check the presence of these fields in the wiki titled 'Experience Event Payload from Campaign'.